bin/faucet/README.md

```diff
@@ -1,6 +1,8 @@
 # Miden faucet
 
-This crate contains a binary for running a Miden testnet faucet.
+This crate contains a binary for running a Miden testnet faucet. 
+
+For comprehensive guides, API reference, and examples, see the [Miden Faucet Documentation](/docs/src/SUMMARY.md).
```

Comment:
I would link this to `https://0xmiden.github.io/miden-faucet`. We will deploy the mdBook in this repo (and I'm not sure we actually need to add it to the general Miden docs yet).